recover_affected_workloads.md

```diff
@@ -21,12 +21,17 @@ For example, in the case where the node fails to update its status to the
 masters the node condition will be set to `type:Ready status:Unknown`.
 
 In order to gain the workload back, reschedule it and start it all over, the
-control plane needs to know that the running node is gone and the only way to achieve
-that is by deleting the node object, which will trigger [scheduling the pods for
-deletion][2]
+control plane needs to know that the running node is gone and there are two ways to do it:
+
+- deleting the node object, which will trigger [scheduling the pods for deletion][2]
+- adding an [`out-of-service` taint][4] on the Node, which triggers pods on the node
```

Comment:
The link number should be 3 as it right after 2.

Reply:
I think it would be just easier to use implicit reference links

Reply:
SGTM I can't recall why it was used this way. NP for changing it from POV as long we are consistent with the rest of the website pages.